project-p2-layouts-master/code/style.css

```diff
@@ -0,0 +1,252 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@300;500;600;900&display=swap');
+
+body {
+  background-color:	rgb(177, 212, 195);
+}
+
+img {
+  width: 60%;
+  border-radius: 50%;
+}
+.center {
+  display: block;
+  margin-left:auto;
+  margin-right: auto; 
+  width: 50%; 
+}
+#mySidenav a {
```

Comment:
We should try to style classes and not id's. If you need to, you can give an element both a class and an id in the HTML:
`<div id="mySideNav" class="my-side-nav">`